README.md

```diff
@@ -64,7 +64,9 @@ It is recommended to flash the replacement CA to /cert/c2.der and use the hackie
 ```
 cc3200tool -p COM3 write_file certs/server/ca.der /cert/c2.der
 ```
-**Beware** The ```blockCheckRemove.310``` and the ```noHide.308``` patch breaks the content passthrough to Boxine. If you are using firmware 3.1.0_BF4 isn't compatible with many patches, except the alt* ones. Please disable them.
+**Beware** The ```blockCheckRemove.310```, ```noCerts.305``` and the ```noHide.308``` patch breaks the content passthrough to Boxine. If you are using firmware 3.1.0_BF4 isn't compatible with many patches, except the alt* ones. Please disable them by removing them in the [```ngCfg.json```](https://github.com/toniebox-reverse-engineering/hackiebox_cfw_ng/wiki/Bootloader#configuration) on the SD card.
+
+To have the patched toniebox booting, stage 2 of the [hackiebox_cfw_ng install](https://github.com/toniebox-reverse-engineering/hackiebox_cfw_ng/wiki/Install#2-bootloader-stage-2) has to be modified, so that only the content of the *sd* folder is copied and the ```ngCfg.json```  selects e.g. ```ofw2``` as the active image with activated alt* patches.
```

Comment:
What do you mean by only copy the sd folder?

Reply:
What do you mean by replacing? You just copy over the contents of the sd folder to the root of your microSD. Then you copy over your mcuimg.bin (ofw bootloader) to sd:/revvox/boot/ng-ofw1.bin

PS: Copying the ofw bootloader (mcuimg.bin) doesn't break anything.

Reply:
Yeah sorry for not being clear enough. Maybe I just followed the instructions to blindly. The process of selecting firmwareslots by holding the big ear while powering the box is still sort of unclear in the documents (at least to me). I guess we could remove the part about the copy process.